__tests__/__packages__/cli-test-utils/src/TestUtils.ts

```diff
@@ -81,8 +81,8 @@ export function isStderrEmptyForProfilesCommand(output: Buffer): boolean {
  */
 export function stripProfileDeprecationMessages(stderr: Buffer | string): string {
     return stderr.toString()
-        .replace(/Warning: The command 'profiles [a-z]+' is deprecated\./g, "")
-        .replace(/Recommended replacement: The 'config [a-z]+' command/g, "")
+        .replace(/Warning: The command 'profiles [a-z\s]+' is deprecated\./g, "")
+        .replace(/Recommended replacement: The 'config [a-z\s]+' command/g, "")
```

Comment:
Instead of `[a-z\s]` should we use `[a-z0-9\s]` or `[\w\s]`?

Currently we don't handle numbers so for a plugin like CA7 that used to offer the command `zowe profiles list ca7` this may fail.

Reply:
that's a good point 🙏 

resolved by 9bea08419b932e421e5f1fd4c3386433267c7774